.idea/compiler.xml

```diff
@@ -10,7 +10,7 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel>
-      <module name="Java-Homework" target="1.8" />
+      <module name="Java-Homework" target="14" />
```

Comment:
You don't have to commit all. This can be simply not selected in the changelist.

Reply:
ok! Sorry.